cfme/infrastructure/datastore.py

```diff
@@ -1,65 +1,123 @@
 """ A model of an Infrastructure Datastore in CFME
-
-
-:var page: A :py:class:`cfme.web_ui.Region` object describing common elements on the
-           Datastores pages.
 """
-from functools import partial
-
 from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic.widget import View, Text
+from widgetastic_patternfly import Dropdown, Accordion, FlashMessages
 
-from cfme.exceptions import CandidateNotFound, ListAccordionLinkNotFound
-from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import (
-    Quadicon, Region, listaccordion as list_acc, toolbar as tb,
-    flash, InfoBlock, match_location, fill, paginator, accordion
-)
-from cfme.web_ui.form_buttons import FormButton
+from cfme.base.login import BaseLoggedInPage
+from cfme.infrastructure.host import HostsView
 from utils.appliance import Navigatable
 from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 from utils.pretty import Pretty
-from utils.providers import get_crud
 from utils.wait import wait_for
+from widgetastic_manageiq import (ManageIQTree, SummaryTable, ItemsToolBarViewSelector,
```

Comment:
done. honestly, I doubt it is correct because widgetastic_manageiq isn't 3d party library. It comes with integration_tests.